pavelib/assets.py

```diff
@@ -957,6 +957,25 @@ def update_assets(args):
             platform_static_dir=EDX_PLATFORM_STATIC_ROOT_BASE
         ))
 
+    print ("Sync static from {static_collector_dir}/{current_sys}/* to {platform_static_dir}/{current_sys}/ with --delete-after option".format(
+            static_collector_dir=django_settings.STATIC_ROOT_BASE,
+            platform_static_dir=django_settings.EDX_PLATFORM_STATIC_ROOT_BASE,
+            current_sys=sys
+        ))
+
+    for sys in args.system:
```

Comment:
We already have this loop, I would use it instead of creating the new one.

Reply:
Done